Assignment01

NGeethaReddy opened the pull request.

NGeethaReddy committed:
Assignment01

ManidharRedd reviewed (commented):
Please dnt commit .class files and .kotlin_module

ManidharRedd commented on Assignment01/src/Arrays/ReversedArray.java:
```diff
@@ -0,0 +1,29 @@
+public class ReversedArray {
+
+    public static void main(String[] args) {
+        Scanner sc = new Scanner(System.in);
```
Please add space below main method

ManidharRedd commented on Assignment01/src/Arrays/ReversedArray.java:
```diff
@@ -0,0 +1,29 @@
+        sample = sc.nextLine();
+        char[] vowels = {'a', 'e', 'i', 'o', 'u'};
+        int vowelsCount = 0;
+        for (int i = sample.length()-1; i >= 0; i--) {
```
Please provide a space below int vowelsCount

ManidharRedd commented on Assignment01/src/Arrays/ReversedArray.java:
```diff
@@ -0,0 +1,29 @@
+        int vowelsCount = 0;
+        for (int i = sample.length()-1; i >= 0; i--) {
+            Character letter = sample.charAt(i);
+            reversedString.append(letter);
```
Please provide a space below  reversedString.append(letter);

ManidharRedd commented on Assignment01/src/Television/TvSelection.java:
```diff
@@ -0,0 +1,34 @@
+            }
+        }
+        public static void main(String[] args) {
+            Scanner sc = new Scanner(System.in);
```
Please add space below main method

ManidharRedd commented on Assignment01/src/Television/TvSelection.java:
```diff
@@ -0,0 +1,34 @@
+            System.out.println("1. Black and White TV");
+            System.out.println("2. Color TV");
+            String choiceInput = sc.nextLine();
+            if (choiceInput.equals("1. Black and White TV")) {
```
Please provide a space below String choiceInput = sc.nextLine();

NGeethaReddy committed:
Delete ReversedArray.class

NGeethaReddy committed:
Update ReversedArray.java

space added below main method at line 9
space added below int vowelsCount at line 16
space added below reversedString.append(letter) at line 21

NGeethaReddy committed:
Delete Assignment01.kotlin_module

NGeethaReddy committed:
Delete TvSelection$BlackAndWhiteTv.class

NGeethaReddy committed:
Delete TvSelection$ColorTv.class

NGeethaReddy committed:
Delete TvSelection.class

NGeethaReddy committed:
Delete Assignment01.iml

NGeethaReddy committed:
Update TvSelection.java

added space below main method at line 18
added space below String choiceInput = sc.nextLine(); at line 24

Commit:
adding Reversedarray

Commit:
updated Reversed Array

Commit:
Added Television

Commit:
Updated Television

NGeethaReddy committed:
Delete Assignment01/src directory
